src/api/controllers/roles.controller.js

```diff
@@ -0,0 +1,77 @@
+import { StatusCodes } from "http-status-codes";
+import models from "../../db/models";
+
+export const getRoles = async (req, res, next) => {
+  try {
+    const roles = await models.role.findAll();
```

Comment:
create new services file in the services folder named it role.services.js and add this code to find roles